packages/blocks/src/page-block/utils/container-operations.ts

```diff
@@ -277,6 +281,19 @@ export function handleSelectAll() {
   resetNativeSelection(range);
 }
 
+export function handleSelectAll(selection: DefaultSelectionManager) {
+  if (selection.state.selectedBlocks.length === 0) {
+    const currentRange = getCurrentRange();
+    const rangeRect = currentRange.getBoundingClientRect();
+    selection.selectBlocksByRect(rangeRect);
+  } else {
+    const rect = document.body.getBoundingClientRect();
```

Comment:
@JimmFly I added this API, which can be easier to work with :)

Reply:
> @JimmFly I added this `selectBlocksByRect` API, which can be easier to work with :)

Oh, that seems more reasonable. Thanks for all you have done.